api/src/queries/administrative-activity/administrative-activity-queries.ts

```diff
@@ -81,6 +81,40 @@ export const getAdministrativeActivitiesSQL = (
   return sqlStatement;
 };
 
+/**
+ * SQL query to get a list of administrative activities, optionally filtered by the administrative activity type name.
+ *
+ * @param {number} [administrativeActivityTypeId]
+ * @returns {SQLStatement} sql query object
+ */
+export const getAdministrativeActivitiesTypeNameSQL = (administrativeActivityTypeId?: number): SQLStatement | null => {
```

Comment:
```suggestion
export const getAdministrativeActivityById = (administrativeActivityTypeId: number): SQLStatement => {
```

Reply:
- Tweak to the name, since we are returning the whole row, not just the type name.
- Made the id a required value, since this wont work without it.
- Removed the `| null` from the return, since we never return null here.